src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Handlers/Label/LabelHandlerTests.cs

```diff
@@ -7,7 +7,7 @@ namespace Xamarin.Platform.Handlers.DeviceTests
 {
 	public partial class LabelHandlerTests : HandlerTestBase<LabelHandler>
 	{
-		[Fact]
+		[Fact(DisplayName = "[LabelHandler] Background Color Initializes Correctly")]
```

Comment:
Should we use [{ControName}] or [{ControlName} Handler] (Example: [Label] or [LabelHandler]?

Reply:
I think just ControlName for now.